pkg/operator/ceph/object/user.go

```diff
@@ -226,3 +231,38 @@ func UnlinkUser(c *Context, id, bucket string) (string, int, error) {
 	}
 	return result, RGWErrorNone, nil
 }
+
+func EnableUserQuota(c *Context, id string) (string, int, error) {
+	logger.Debug("Enabling user quota for %q", id)
+	args := append([]string{"quota", "enable", "--quota-scope", "user", "--uid", id})
+	result, err := runAdminCommand(c, args...)
+	if err != nil {
+		if strings.Contains(result, "uid is required for quota operation") {
+			return "", RGWErrorBadData, errors.Wrapf(err, "invalid user name")
+		}
+		if strings.Contains(result, "failed updating user info") {
+			return "", RGWErrorUnknown, errors.Wrapf(err, "setting quota failed")
+		}
+	}
+	return result, RGWErrorNone, err
+
+}
+func SetQuotaUserObjectMax(c *Context, id string, maxobjects string) (string, int, error) {
```

Comment:
nit: newline between the functions and also add a comment to each public function.

Reply:
will address in next PR

Reply:
Done

Reply:
Done